### PR TITLE
docs(intergration): examples

### PR DESCRIPTION
**Pull request by fforootd**
This adds the following examples

- Grafana
- OAuth Proxy
- Ambassador
- Typescript

### CODE REVIEW

**Comment by mffap** [comment]
duplicate of #944

File: site/docs/administrate/09-authorizations.en.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+title: Authorizations
+---
+
+### ZITADEL's management Roles
+
+ZITADEL's own role model is built around the IAM resource. The roles have some hierarchies to them. For example a IAM_OWNER can view and edit every resource of the system. ORG_OWNERS can only manage their resources included within their organisation. This includes projects, clients, users, and so on.
+
+#### How to give a user ZITADEL Roles
+
+With ZITADEL Conso
```
Comment (fforootd) [inline site/docs/administrate/09-authorizations.en.md:11]:
```suggestion
```

File: site/docs/integrate/00-overview.en.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ description: ...
 
 > All documentations are under active work and subject to change soon!
 
-### Application Integration
+This Integration guide gives you recommendations on how to intergrate different Clients with **ZITADEL**
```
Comment (fforootd) [inline site/docs/integrate/00-overview.en.md:8]:
```suggestion
This Integration guide gives you recommendations on how to integrate different Clients with **ZITADEL**
```

File: site/docs/integrate/05-proxy.en.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  rules:
+    - host: "*"
+      path: /backend/get-quote/
+      filters:
+        - name: zitadel-filter
+```
+
+#### Use Ambassador to check JWT Bearer Tokens
+
+If you would like **Ambassador** to verify a JWT token from the autorization header you can do so by configuring **ZITADEL's** endpoints.
```
Comment (fforootd) [inline site/docs/integrate/05-proxy.en.md:53]:
```suggestion
If you would like **Ambassador** to verify a JWT token from the authorization header you can do so by configuring **ZITADEL's** endpoints.
```

**Review by mffap — APPROVED** [review]

**Pull request merged**